### PR TITLE
PhantasmicDream Fluff Item: Changeling Hat

### PR DESCRIPTION
**Pull request by tigercat2000**

### CODE REVIEW

**Comment by SamHPurp** [comment]
A fluff thats actually 12 fluffs?

**Comment by FreeStylaLT** [comment]
👌

**Comment by Crazylemon64** [comment]
stuncuff it and toss it in the cremator STAT /s

👌

**Comment by PhantasmicDream** [comment]
Golly gee, I didn't think it'd be thrown on here already!!  ^-^

**Comment by taukausanake** [comment]
-1/10 needs more hats

File: code/modules/customitems/item_defines.dm [hunk 1]
```diff
@@ -697,3 +697,32 @@
...
+	options["Bunny"] = "bunny_witch"
+	options["Potions"] = "potions_witch"
+	options["Syndicate"] = "syndie_witch"
+	options["Nanotrasen"] ="nt_witch"
+
+	var/choice = input(user, "To what form do you wish to Shapeshift this hat?", "Shapeshift Hat") as null|anything in options
+
+	if(choice && !user.stat && in_range(user, src))
+		icon_state = options[choice]
+		to_chat(user, "Your strange witch hat has now shapeshifted into it's [choice] form!")
```
Comment (FlattestGuitar) [inline code/modules/customitems/item_defines.dm]: its*

**Comment by KingPhilipIII** [comment]
I can't be held responsible if I arrest your hat and have it spaced.

**Comment by Fox-McCloud** [comment]
@Necaladun @Crazylemon64 @FreeStylaLT 

needs added to the database

**Pull request merged**